__tests__/app.test.js

```diff
@@ -71,6 +71,89 @@ describe("GET /api/reviews", () => {
         expect(reviews.length).toBe(13);
       });
   });
+  test("200: accepts a category query", () => {
+    return request(app)
+      .get("/api/reviews?category=euro+game")
+      .expect(200)
+      .then(({ body: reviewsByCategory }) => {
+        reviewsByCategory.forEach((review) => {
+          expect(review).toMatchObject({
+            title: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_img_url: expect.any(String),
+            review_body: expect.any(String),
+            category: "euro game",
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("404: should test if category is not found", () => {
+    return request(app)
+      .get("/api/reviews?category=euro")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+  test("404: should test if query category is spelt wrong ", () => {
```

Comment:
This and the previous test are checking for the same thing, we can delete one of this 🙂